src/interface/src/styleguide/search-bar/search-bar.component.html

```diff
@@ -4,13 +4,16 @@
   #origin="matAutocompleteOrigin">
   <sg-input-field leadingIcon="search" [disabled]="false">
     <input
+      #inputElement
       sgInput
       placeholder="{{ searchPlaceholder }}"
       [matAutocomplete]="searchHistory"
       [matAutocompleteConnectedTo]="origin"
       (input)="onSearchInputChange($event)" />
     <mat-autocomplete #searchHistory="matAutocomplete" panel="acpanel">
-      <div class="autocomplete-label">{{ autocompleteTitle }}</div>
+      <div class="autocomplete-label" *ngIf="historyItems.length">
```

Comment:
adding this ngIf to the label, because it stays on the CDK layer, even when there's no autocomplete, obscuring the UI for anything underneath.
I'm wondering if we should have an explicit Input() option for allowing autocomplete or not, rather than implicitly using historyItems here.